### ISSUE TITLE
Install tool: TypeError: $(...) is null

### ISSUE THREAD

**Issue by ghost** [contao/installation-bundle#19]
<a href="https://github.com/Fuzzyma"><img src="https://avatars.githubusercontent.com/u/3540542?v=3" align="left" width="42" height="42"></img></a> [Issue](https://github.com/contao/core/issues/8386) by @Fuzzyma
June 29th, 2016, 16:06 GMT

Iam using contao 4.2. When running the install tool, I get a javascript error in Firefox:

```
TypeError: $(...) is null
Theme.setupMenuToggle() hover.js:2
<anonym> hover.js:2
.fireEvent/<() mootools.min.js:3
forEach() self-hosted:232
forEach() self-hosted:243
.each() mootools.min.js:2
.fireEvent() mootools.min.js:3
h() mootools.min.js:4
1 hover.js:2:2766
```

**Comment by ghost** [comment]
<a href="https://github.com/leofeyer"><img src="https://avatars.githubusercontent.com/u/1192057?v=3" align="left" width="42" height="42"></img></a> Comment by @leofeyer
June 29th, 2016, 19:26 GMT

Does not happen on my system. Anyone else?

**Comment by ghost** [comment]
<a href="https://github.com/xchs"><img src="https://avatars.githubusercontent.com/u/1218766?v=3" align="left" width="42" height="42"></img></a> Comment by @xchs
June 29th, 2016, 22:04 GMT

> Anyone else?

Yep. I can reproduce it on my system (FF, Win).

**Comment by leofeyer** [comment]
Fixed in contao/core-bundle@f5f26cb8fcb346fc1fd2b93db8e53a3e56b31f09.
